game/state/rules/weeklyrating.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+namespace OpenApoc
+{
+
+class WeeklyRating
+{
+  public:
+	enum class Type
+	{
+		Terrible,     // -1600
```

Comment:
Is it really necessary to have the names as part of the rating enum? Would just an integer be better and more flexible in the future? And just map them to names for user-visible text output?